sdk/src/main/scala/io/horizen/account/mempool/MempoolMap.scala

```diff
@@ -59,15 +60,21 @@ class MempoolMap(
     if (!contains(ethTransaction.id)) {
       val account = ethTransaction.getFrom
 
-      //Reject transactions that are too big
+      // Reject transactions that are too big
       val txSize = ethTransaction.size()
       if (txSize > MaxTxSize) {
         log.trace(s"Transaction $ethTransaction size exceeds maximum allowed size: current size $txSize, " +
           s"maximum size: $MaxTxSize")
         throw TxOversizedException(account.asInstanceOf[AddressProposition].address(), txSize)
       }
 
-      //Reject transactions with a nonce gap too big
+      // Reject transaction with max fee less than the current minimum base fee
+      val currentEpochNumber = baseStateReaderProvider.getBaseStateReader().getConsensusEpochNumber.getOrElse(0)
```

Comment:
There is another similar check related to the gas limit of the transaction based on a parameter taken from the ForkManager inside the AccountState (https://github.com/HorizenOfficial/Sidechains-SDK/blob/dev/sdk/src/main/scala/io/horizen/account/state/AccountState.scala#L469). Have you considered to add the additional check here instead of the Mempool?

Reply:
Good point. Actually, a change in the `validate` method is necessary, otherwise a custom node could still include TXs with lower-than-minimum fees in a block.